reference/docs-conceptual/community/contributing/editorial-checklist.md

```diff
@@ -56,8 +57,8 @@ articles in the Contributor's Guide for detailed explanations and examples of th
 - Blank line before and after
 - Use tagged code fences - **powershell**, **Output**, or other appropriate language ID
 - Untagged fence - syntax blocks or other shells
-- Put **Output** in separate code block except for simple examples where you don't intend the for
-  the reader to use the **Copy** button
+- Put output in separate code block except for simple examples where you don't intend the for the
+  reader to use the **Copy** button
 - See list of [supported languages](/contribute/code-in-docs#supported-languages)
```

Comment:
Link for "See list of supported languages" gives a 404 error.

Reply:
This is normal in the preview when the link is in an different repo.